sunbeam-python/sunbeam/commands/prepare_node.py

```diff
@@ -111,19 +111,66 @@
 fi
 """
 
+BOOTSTRAP_TEMPLATE = f"""
+# Install the lxd snap
+sudo snap install lxd --channel {LXD_CHANNEL}
+USER=$(whoami)
+# Ensure current user is part of the LXD group
+sudo usermod --append --groups lxd $USER
+
+# Try to determine if LXD is already bootstrapped
```

Comment:
If lxd is already bootstrapped, need to error out if the bridge is lxdbr0. Maybe good to add further check if juju is already bootstrapped to make the script idempotent.